### PR TITLE
README: Fix formatting and convert to MD

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
Tests fail because of https://github.com/sociomantic-tsunami/cachalot/issues/76, but the formatting looks correct now, and more clear.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+If a TAG is specified but is a pre-release (as per SemVer, i.e. it contains a -), the default tagged image will be ORG/IMG:DIST-vX-test (instead of ORG/IMG:DIST-vX) and ORG/IMG will not be tagged at all.
+
+make push will push all the images that would be generated by make build (same rules apply for TAG).
+
+Docker Images Publishing
+========================
+
+Images are automatically published for tags via [Travis](https://travis-ci.org/sociomantic-tsunami/cachalot) to [DockerHub](https://travis-ci.org/sociomantic-tsunami/cachalot/settings)'s [sociomantictsunami](https://cloud.docker.com/u/sociomantictsunami/repository/list) organization.
+
```
Comment (daniel-zullo) [inline README.md:22]: I've got 404  (page not found) on the last link), is the correct link?

**Review by daniel-zullo — COMMENTED** [review]
One question and LGTM

File: README.md (hunk 1 above)
Comment (leandro-lucarella-sociomantic) [reply README.md:22]: Yes, you don't have permissions to access the settings page, we probably have to change that.

**Comment by leandro-lucarella-sociomantic** [comment]
Fixed wrong link.

**Pull request merged**